ckanext/s3filestore/click_commands.py

```diff
@@ -81,3 +81,34 @@ def upload_resources():
             len(uploaded_resources)),
         fg=u'green',
         bold=True)
+
+
+@click.command(u's3-test-connection', short_help=u'Test connection to AWS S3')
```

Comment:
Perhaps there should be an `s3` command group?

Reply:
I agree with you, @ThrawnCA. Probably a `s3filestore` group